cmd/createcluster.go

```
@@ -168,6 +167,10 @@ func runCreateCluster(w io.Writer, conf clusterConfig) error {
 		}
 	}
 
+	if err := validateClusterConfig(conf); err != nil {
```

fail fast: do validation at the start